abz/acousticbrainz.py

```diff
@@ -91,7 +97,14 @@ def submit_features(recordingid, features):
 def process_file(filepath):
     _start_progress(filepath)
     if is_processed(filepath):
-        _update_progress(filepath, ":) done", GREEN)
+        processed_status = get_processed_status(filepath)
+        if processed_status == None:
+	        status = ":) done"
+	        colour = GREEN
+        else:
+	        status = ":( nombid"
```

Comment:
There are other reasons for failing than "nombid", as @alastair also mentions in his comment. I get this when I run `SELECT reason FROM "main"."filelog" WHERE reason <> 'NULL';` on my local abzsubmit db:
```
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"nombid"
"extractor"
"extractor"
"extractor"
"nombid"
"nombid"
```

There might be other errors too.